### PR TITLE
More ciqual additions

### PR DESCRIPTION
**Pull request by aleene**
From line 1662 onward

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -19073,7 +19240,11 @@ fr:huile de palme totalement hydrogénée, palme totalement hydrogénée
 hu:teljes mértékben hidrogénezett pálmaolaj, teljesen hidrogénezett pálmaolaj, teljes mértékben hidrogénezett pálmamag-olaj
 it:Olio di palma idrogenato
 nl:geheel geharde palm
+<<<<<<< HEAD
```
Comment (teolemon) [inline taxonomies/ingredients.txt:19243]: Merge marks @aleene

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -19073,7 +19240,11 @@ fr:huile de palme totalement hydrogénée, palme totalement hydrogénée
...
+=======
 ro:unt de palmier complet hidrogenat
+>>>>>>> 0bae371ce76d1b59d5d0bdd392423e1de4526d23
+sl:popolnoma hidrogenirano palmino olje
```
Comment (teolemon) [inline taxonomies/ingredients.txt:19247]:
```suggestion
ro:unt de palmier complet hidrogenat
sl:popolnoma hidrogenirano palmino olje
```

**Comment by teolemon** [comment]
Some merge conflicts with 'main'

**Comment by teolemon** [comment]
Merge conflict to solve @aleene

**Comment by teolemon** [comment]
@aleene merge conflicts

**Comment by aleene** [comment]
Seems all is good now

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -17041,58 +17076,81 @@ fr:os de porc
...
-
-<en:pork belly
-en:raw pork belly 
-fr:poitrine de porc crue
 ciqual_food_code:en:28002
 ciqual_food_name:en:Pork, belly, raw
 ciqual_food_name:fr:Porc, poitrine, crue
 # TODO: investigate between pork belly and pork breast
+description:de:Als Schweinebauch werden die gut mit Fett durchwachsenen und mit Rippen durchzogenen Stücke der hinteren (caudalen), unteren (ventralen) Brustkörbe der Hausschweine bezeichnet.
+description:en:Pork belly is a boneless and fatty cut of meat[2] from the belly of a pig.
```
Comment (teolemon) [inline taxonomies/ingredients.txt:17135]:
```suggestion
description:en:Pork belly is a boneless and fatty cut of meat from the belly of a pig.
```

File: taxonomies/ingredients.txt [hunk 4]
```diff
@@ -19240,10 +19345,6 @@ fr:huile de palme totalement hydrogénée, palme totalement hydrogénée
 hu:teljes mértékben hidrogénezett pálmaolaj, teljesen hidrogénezett pálmaolaj, teljes mértékben hidrogénezett pálmamag-olaj
 it:Olio di palma idrogenato
 nl:geheel geharde palm
```
Comment (teolemon) [inline taxonomies/ingredients.txt:19347]:
```suggestion
nl:geheel geharde palm
ro:unt de palmier complet hidrogenat
```

File: taxonomies/ingredients.txt [hunk 5]
```diff
@@ -17204,59 +17262,80 @@ nl:varkensbindweefsel
...
-# sgs:Taukā
-# sr-el:Svinjska mast
-# szl:Tuste
+# pdc:schmals
+# scn:nzunza
+# sgs:taukā
+# szl:tuste
 # yue:豬油
-wikidata:en:Q72827
+wikidata:en:q72827
```
Comment (teolemon) [inline taxonomies/ingredients.txt:17334]:
```suggestion
wikidata:en:Q72827
```

File: taxonomies/ingredients.txt [hunk 6]
```diff
@@ -22825,27 +22976,35 @@ en:Marsala wine
...
-pt:Vinho Marsala
+pt:Vinho Marsala, vinho de Marsala
 ro:Vin de Marsala
 ru:Марсала
 sh:Marsala
+sk:Marsala, víno Marsala
 sv:Marsalavin
 uk:Марсала
+xx:Marsala
+# da,de,es,fifr,hr,hu,lt,nb,nl,pl,sh
```
Comment (teolemon) [inline taxonomies/ingredients.txt:23007]:
```suggestion
# da,de,es,fi,fr,hr,hu,lt,nb,nl,pl,sh
```

Comment (aleene) [reply taxonomies/ingredients.txt:23007]: just comments

File: taxonomies/ingredients.txt [hunk 7]
```diff
@@ -57489,24 +58211,23 @@ ro:cardamon
...
 yi:קארדעמאן
 # arc:ܫܘܫܡܢܝܬܐ
 # arz:حبهان
-# ast:Cardamomo
+# ast:cardamomo
 # gom:वेलची
-# nds:Kardamom
+# nds:kardamom
 # new:एला
 # tcy:ಏಲಕ್ಕಿ
```
Comment (teolemon) [inline taxonomies/ingredients.txt:58230]:
```suggestion
# tcy:ಏಲಕ್ಕಿ
wikidata:en:Q14625808
```

File: taxonomies/ingredients.txt [hunk 8]
```diff
@@ -58669,90 +59589,106 @@ sv:dillsmör
...
+
+<en:rosemary
+en:fresh rosemary
+fr:romarin frais
+ciqual_food_code:en:11068
+ciqual_food_name:en:Rosemary, fresh
+ciqual_food_name:fr:Romarin, frais
+
+<en:rosemary
+endDried rosemary
```
Comment (teolemon) [inline taxonomies/ingredients.txt:59686]:
```suggestion
en:Dried rosemary
```

File: taxonomies/ingredients.txt [hunk 9]
```diff
@@ -59305,80 +60245,84 @@ nl:dragonblad
...
-fr:thym bio
-it:timo bio
-nl:biologische tijm
+ciqual_food_code:en:11070
+ciqual_food_name:en:Thyme, fresh
+ciqual_food_name:fr:Thym, frais
+# 7086 in 18 @2021-09-16
+#ciqual_food_code:en:11038
+#ciqual_food_name:en:Thyme, dried
+#ciqual_food_name:fr:Thym, séché
```
Comment (teolemon) [inline taxonomies/ingredients.txt:60293]: mmh

File: taxonomies/ingredients.txt [hunk 10]
```diff
@@ -59305,80 +60245,84 @@ nl:dragonblad
...
-en:organic thyme
-de:Bio Thymian
-fi:luomu timjami
-fr:thym bio
-it:timo bio
-nl:biologische tijm
+ciqual_food_code:en:11070
+ciqual_food_name:en:Thyme, fresh
+ciqual_food_name:fr:Thym, frais
+# 7086 in 18 @2021-09-16
```
Comment (teolemon) [inline taxonomies/ingredients.txt:60290]:
```suggestion
# 7086 in 18 @2021-09-16
# TODO:split
```

File: taxonomies/ingredients.txt [hunk 11]
```diff
@@ -59811,77 +60755,80 @@ ciqual_food_name:fr:Cerfeuil, frais
...
 # zh-hk:鼠尾草
 # zh-tw:鼠尾草
+#plant:
 wikidata:en:Q1111359
 wikipedia:en:https://en.wikipedia.org/wiki/Salvia_officinalis
-# ingredient/fr:sauge has 175 products in 5 languages @2018-10-10
+ciqual_food_code:en:11069
+ciqual_food_name:en:Sage, fresh
+ciqual_food_name:fr:Sauge, fraîche
+# 1379 in 15 @2021-09-16
```
Comment (teolemon) [inline taxonomies/ingredients.txt:60818]:
```suggestion
# 1379 in 15 @2021-09-16
# TODO: split
```

**Review by teolemon — APPROVED** [review]

**Pull request merged**